src/components/DpEditor/libs/editorMention.js

```diff
@@ -1,11 +1,13 @@
 import { mergeAttributes } from '@tiptap/core'
-import { Mention } from '@tiptap/extension-mention'
+// THIS IS A WORKAROUND.
+// In our setup per default the cjs-file is used. That file for some reason has a problem to resolve an internal dependency (Suggestion) properly.
+import { Mention } from '../../../../node_modules/@tiptap/extension-mention/dist/index.js'
 
 export default Mention.extend({
-  renderHTML({ node, HTMLAttributes }) {
+  renderHTML({ node, options }) {
     return [
       'span',
-      mergeAttributes({ 'data-type': this.name, 'data-suggestion-id': node.attrs.id }, this.options.HTMLAttributes, HTMLAttributes),
+      mergeAttributes({ 'data-type': this.name, 'data-suggestion-id': node.attrs.id }, options.HTMLAttributes),
```

Comment:
Why can `this.options.HTMLAttributes` be removed? What is `this.options` anyway?

Reply:
https://github.com/ueberdosis/tiptap/blob/7567acebd62a23a2820d67a500bc5009a1099bdb/packages/extension-mention/src/mention.ts#L20

Reply:
I rewrote the logic and removed the file.
Please have another look:
916a68c8951499dc488b5abf48acb3d360474ab9